cmd/edgectl/aes_install.go

```diff
@@ -20,15 +20,23 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/datawire/ambassador/pkg/k8s"
-	"github.com/datawire/ambassador/pkg/supervisor"
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	k8sTypesMetaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	k8sClientCoreV1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/rest"
+
+	"github.com/datawire/ambassador/pkg/helm"
+	"github.com/datawire/ambassador/pkg/k8s"
+	"github.com/datawire/ambassador/pkg/supervisor"
 )
 
+// clusterInfo describes some properties about the cluster where the installation is performed
+type clusterInfo struct {
```

Comment:
I would move this struct definition towards the bottom of this file. It seems to be the style here as other structs are defined below.